openshift/template.yaml

```diff
@@ -5,304 +5,216 @@ labels:
 metadata:
   name: thoth-core
   annotations:
-    description: thoth-core
+    description: This is Thoth Core - Analyzer and Results API and supporting components 
+    openshift.io/display-name: Thoth Core
+    tags: poc,thoth,ai-stacks
+    template.openshift.io/documentation-url: https://github.com/AICoE/Thoth/
+    template.openshift.io/long-description: This template defines resources needed
+      to deploy Thoth Core Services as a Proof-of-Concept to OpenShift.
+    template.openshift.io/provider-display-name: Red Hat, Inc.
 
 objects:
+
 #
-# Secrets
-#
+# Application Configuration
+## Secrets
 - apiVersion: v1
   kind: Secret
   metadata:
-    name: secret
-    namespace: ${THOTH_FRONTEND_NAMESPACE}
```

Comment:
Why was namespace deleted? Will these changes made for the transition to configmaps work when we will deploy thoth to multiple namespaces as it is designed?

Reply:
thats right, secrets are only accessible from within the same namespace, I gonna fix it!